deploy/manifests/00-crds.yaml

```diff
@@ -5351,4 +5340,3 @@ status:
     plural: ""
   conditions: []
   storedVersions: []
----
```

Comment:
hm, removing this line is going to cause an issue when we generate 'static manifests' I think - this file is concatted to the `01-namespace.yaml` file (you can see here: https://github.com/jetstack/cert-manager/blob/master/deploy/manifests/01-namespace.yaml).

You can see where the release tool assembles these here: https://github.com/jetstack/cert-manager/blob/master/hack/release/pkg/build/manifests/plugin.go#L133-L137

I think the simplest solution here is to simply add `---` to the start of the `01-namespace.yaml` file...

Reply:
This is also a good prompt for us to set up end-to-end tests that utilise these static manifests.. currently all our tests run using the Helm chart!

Reply:
> I think the simplest solution here is to simply add --- to the start of the 01-namespace.yaml file...

Yes…, and remove the last blank line. I'll do that.

FYI: https://yaml.org/spec/1.2/spec.html#id2760395